_posts/2019-05-08-model-serving-in-pyorch.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: blog_detail
+title: 'Model Serving in PyTorch'
+author: Jeff Smith
+redirect_from: /2019/05/08/model-serving-in-pyorch.html
+---
+
+# Model Serving in PyTorch
```

Comment:
We can remove this title since it's duplicated above and will show weirdly in the preview.

Reply:
Gotcha. Done.